### PR TITLE
[FLINK-14031][examples][table]Added the blink planner dependency and …

### PR DESCRIPTION
**Pull request by GatsbyNewton**
## What is the purpose of the change
Added the blink planner dependency and examples which use blink planner to flink-examples-table.

## Brief change log
- Add the blink planner dependency.
- Add examples which use blink planner.

## Verifying this change
Manually verified. Ran examples on idea and a local cluster.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by GatsbyNewton** [comment]
@KurtYoung This is a new PR. Could you please kindly help to review the code changes? Thank you.

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java [hunk 1]
```diff
@@ -42,9 +47,24 @@
...
 		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-		StreamTableEnvironment tEnv = StreamTableEnvironment.create(env);
+		StreamTableEnvironment tEnv;
+		if (Objects.equals(planner, "blink")) {	// use blink planner in streaming mode
+			EnvironmentSettings settings = EnvironmentSettings.newInstance()
+				.useBlinkPlanner()
+				.inStreamingMode()
+				.build();
+			tEnv = StreamTableEnvironment.create(env, settings);
+		} else if (Objects.equals(planner, "flink")){	// use flink planner in streaming mode
```
Comment (KurtYoung) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java]: nit: space before `{`

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java [hunk 2]
```diff
@@ -42,9 +47,24 @@
...
+			tEnv = StreamTableEnvironment.create(env);
+		} else {
+			System.err.println("The planner is incorrect. Please use --planner to specify planner type.");
```
Comment (KurtYoung) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java]: Could you explain it more clearly in this error message that you have use either `flink` planner or `blink` planner?

Comment (KurtYoung) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java]: The user will not have such knowledge when they are trying the examples without looking into the codes.

**Comment by KurtYoung** [comment]
LGTM, merging this..

**Pull request closed without merging**